include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +316,13 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     *
+     */
+    template <class InputVector>
+    void get_function_values_from_local_dof_values (const InputVector &dof_values,
+                                                    std::vector<typename OutputType<typename InputVector::value_type>::value_type> &values) const;
```

Comment:
I see. That makes sense. I suspect that few users will use the `OutputType` class and instead indeed just work out what the correct type should be (I suspect -- but don't know -- that that should be easy), but it's 12 or a dozen, so just go with whatever you already have!